### ISSUE TITLE
Where clause inconsistent in v2

### ISSUE THREAD

**Issue by cipriancaba** [maoosi/prisma-appsync#46]
I have the following model in `schema.prisma`

```
model WarehouseStockBatch {
  id                Int                @id @default(autoincrement())
  quantity      Int               @default(0)
  sku               String
  stockOrderNumber  String

  @@unique([sku, stockOrderNumber])
}
```
I am trying to use an atomic operation on the batch, but the mutation fails even with the regular updates (data). The reason is `Argument where of type WarehouseStockBatchWhereUniqueInput needs exactly one argument, but you provided sku and stockOrderNumber.\\nUnknown arg `sku` in where.sku for type WarehouseStockBatchWhereUniqueInput. Did you mean `id`?\\nUnknown arg `stockOrderNumber` in where.stockOrderNumber for type WarehouseStockBatchWhereUniqueInput. Did you mean `sku_stockOrderNumber`?\\n\",\"type\":\"INTERNAL_SERVER_ERROR\",\"code\":500}`

Basically, I am trying to target the index created for the update. This works very well with the prisma client, but for some reason the graphql generated where clause is not the same as in Prisma. The one from graphql:
```
input WarehouseStockBatchWhereUniqueInput {
    id: Int
    sku: String
    stockOrderNumber: String
}
```

and the one from prisma:
```
export type WarehouseStockBatchWhereUniqueInput = {
    id?: number
    sku_stockOrderNumber?: WarehouseStockBatchSkuStockOrderNumberCompoundUniqueInput
  }
```
You can notice the missing `sku` and `stockOrderNumber` params in prisma and also the unique index being present.

**Comment by maoosi** [comment]
@cipriancaba I've just release [1.0.0-rc.1](https://github.com/maoosi/prisma-appsync/releases/tag/1.0.0-rc.1). Please give it a try!

I also recommend checking out the new local dev environment included when running `npx create-prisma-appsync-app` on a new project - it can prove useful when developing with Prisma-AppSync.
